tree/dataframe/inc/ROOT/RDF/InterfaceUtils.hxx

```diff
@@ -271,20 +273,40 @@ BuildAction(const ColumnNames_t &colNames, const std::shared_ptr<SnapshotHelperA
       isDefine[i] = colRegister.IsDefineOrAlias(colNames[i]);
 
    std::unique_ptr<RActionBase> actionPtr;
-   if (!ROOT::IsImplicitMTEnabled()) {
-      // single-thread snapshot
-      using Helper_t = SnapshotHelper<ColTypes...>;
+   if (snapHelperArgs->fToNTuple) {
+#ifdef R__HAS_ROOT7
+      // TODO(fdegeus) Add MT snapshotting
```

Comment:
Do we want to just throw an exception for now informing the user that Snapshot to RNTuple with IMT enabled is not supported?